hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -217,6 +217,7 @@ def _prepare_holidays_meeting_values(self):
             "user_id": SUPERUSER_ID,
             "privacy": "confidential",
             "show_as": "busy",
+            "partner_ids": False,
```

Comment:
It seems this should be:

```suggestion
            "partner_ids": [],
```

Reply:
Right!